### ISSUE TITLE
A way to solve merge conflicts

### ISSUE THREAD

**Issue by MuratUrsavas** [soramimi/Guitar#21]
Hi,

I'm in search of a decent Git GUI and found this repository. As I'm a Qt developer myself I was able to build and run the software. It was a bit clumsy but OK. (I'm thinking to fix them via sub-repos later for zlib and openssl)

Anyway, my problem is I want to do a big merge between two branches and having expected conflicts. But couldn't find a way to trigger a merge tool (e.g. Kdiff3) for that particular file. Am I missing something or this feature is really missing.

I really don't have extra time nowadays, but seriously considering to contribute this project in the future.

Thanks for your great job.

**Comment by soramimi** [comment]
Currently, the Guitar does not support collaboration with merge tools. I also want a good merge tool, so I would like to try making it myself if possible. Of course, it will be supported for multiple platforms. If the user wishes, I would like to be able to launch external tools as well. After the Guitar version 1 is released, I would like to develop the best marge tool :-)

Your contribution is very welcome.
Thank you.

**Comment by MuratUrsavas** [comment]
Yeah, it would be good to have new merge tool, but as a starter I'd like to have an working existing external one :) Since Kdiff3 is my favorite, I'll try to add this feature.

Like I said, I'm not a git expert but would like to contribute somehow.

Speaking about the v1.0, do you have a written road map or feature backlog? We can have an idea what to expect next.

**Comment by soramimi** [comment]
I am not a Git expert as well. Because I started this project to master Git myself.

I hope more people will use this program. So, English document is necessary for this project.

Even you may know, many Japanese are not good at English. Me too. It is impossible only with the Google translation service.

I want many opinions and use cases.
And I want to learn more Git and English.

**Comment by MuratUrsavas** [comment]
I hope I can help you with that.
